media_driver/linux/common/ddi/wayland-drm-client-protocol.h

```diff
@@ -0,0 +1,307 @@
+/* Generated by wayland-scanner 1.12.0 */
```

Comment:
If this is autogenerated, wouldn't it be left out of git and generate it within the build system?

Reply:
yes, agree. will change it , but now will keep it for test purpose.